### ISSUE TITLE
TASK: Render Rawhide builds tagged into ELN differently in the status report

### ISSUE THREAD

**Issue by hroncok** [fedora-eln/eln#74]
### What does the ELN SIG need to do?

Since the "new" way of how ELN solves build order issues, all ELN packages are green at https://docs.fedoraproject.org/en-US/eln/status/ -- I think it would make sense to paint the `.fcX` builds in a different color (and change the legend accordingly).

Thanks.

**Comment by sgallagher** [comment]
Agreed, the logic here needs to be updated. Thanks for catching that.

@tdawson Would you look into this, please?

**Comment by tdawson** [comment]
Sorry, this totally fell through the cracks.
I totally agree that the fcX packages should be a different color / state.
I'm more concerned that the status hasn't ran since Nov. 11.
I believe the problem is that the status page update was tied to the eln-periodic check, which has been turned off.
I don't want to turn eln-periodic back on, so I'll look at creating a new jenkins run that just does the check.

**Comment by sgallagher** [comment]
Ah, right. The eln-periodic implementation got migrated into DistroBuildSync.

**Comment by tdawson** [comment]
Turns out that eln-periodic was still running every hour, and failling since Nov. 11.
eln-periodic called eln-build-pipeline, which did the package builds.  eln-build-pipeline was disabled/broken so it would not run, back when Stephen migrating things to DistroBuildSync.
I removed that section (calling eln-build-pipeline) f rom the eln-periodic jenkins file, and now it is running, and the page is getting updated.

Next up, update it so the .fcX are a different color and label.
But that isn't going to happen today, so I thought I'd update it with what has happened thus far.
